README.rst

```diff
@@ -59,7 +59,7 @@ Features
 
 - **Device independent**. The same quantum circuit model can be **run on different backends**
 
-- **Large plugin ecosystem**. Install plugins to run your computational circuits on more devices, including Strawberry Fields and ProjectQ
+- **Large plugin ecosystem**. Install plugins to run your computational circuits on more devices, including Strawberry Fields, ProjectQ, and IBMQ
```

Comment:
qiskit is by IBM, but they do seem to market it as a pretty independent thing. If you look at https://qiskit.org/ you really have to search hard to find an IBM logo somewhere :-) Maybe it would be good to reflect that in the way we refer to it in PennyLane and also simply call it qiskit here?

Reply:
```suggestion
- **Large plugin ecosystem**. Install plugins to run your computational circuits on more devices, including Strawberry Fields, ProjectQ, and Qiskit
```